app/services/scoring/player_svc.rb

```diff
@@ -0,0 +1,7 @@
+class Scoring::PlayerSvc
+    #calculates the points earned by player in match
+    def self.call(player, match)
+        only_stats = PlayerStat.where(match_id: match.id, player_id: player.id).pluck(:stat_id)
+        Stat.where(id: only_stats).sum(:points)
+    end
+end
```

Comment:
i have no idea why this is in this pull request, it should be in the rails-32 one only

Reply:
well to be honest i probably do know why. an unfortunate mistake